src/StudioCore/Assets/Paramdex/ER/Upgrader/massedit.txt

```diff
@@ -48,4 +48,529 @@ param ToughnessParam: added: unk1: = 1
 param ToughnessParam: modified && !added: unk1: = vanillafield unk1
 11001000L
 1.10 - (ToughnessParam) Set unk2 to Vanilla v1.10 values
-param ToughnessParam: modified && !added: unk2: = vanillafield unk2
\ No newline at end of file
+param ToughnessParam: modified && !added: unk2: = vanillafield unk2
+11210015L
+1.12.1 - (WorldMapPointParam) Set dispMask02 to Vanilla v1.12.1 values
+param WorldMapPointParam: modified && !added: dispMask02: = vanillafield dispMask02
+11210015L
+1.12.1 - (WorldMapPointParam) Set pad2_0 to Vanilla v1.12.1 values
+param WorldMapPointParam: modified && !added: pad2_0: = vanillafield pad2_0
+11210015L
+1.12.1 - (WorldMapPointParam) Set unknown_0xb7 to Vanilla v1.12.1 values
+param WorldMapPointParam: modified && !added: unknown_0xb7: = vanillafield unknown_0xb7
```

Comment:
we can coalesce as unknown_0xb[789] if we use = vanilla; (I think previous commands may have been written before that shortcut)